galpy/potential/MovingObjectPotential.py

```diff
@@ -154,8 +159,11 @@ def _phiforce(self,R,z,phi=0.,t=0.):
         #Cylindrical distance
         Rdist = _cylR(R,phi,self._orb.R(t),self._orb.phi(t))
         # Difference vector
-        (xd,yd,zd) = _cyldiff(self._orb.R(t), self._orb.phi(t), self._orb.z(t),
+        orbz = self._orb.z(t) if self._orb.dim() == 3 else 0
+        (xd,yd,zd) = _cyldiff(self._orb.R(t), self._orb.phi(t), orbz,
             R, phi, z)
+        x = self._orb.R(t)*nu.cos(self._orb.phi(t))
```

Comment:
@turnergarrow: are ``x`` and ``y`` as defined in this line used anywhere?

Reply:
No, I think those lines are left over from testing. They can be removed.